src/tests/ftest/util/daos_utils.py

```diff
@@ -92,7 +92,7 @@ def container_create(self, pool, sys_name=None, cont=None, path=None, cont_type=
         """
         # Default to RANK fault domain (rd_lvl:1) when not specified
         if properties:
-            if 'rd_lvl' not in properties:
+            if ('rd_lvl' not in properties) and ('rf_lvl' not in properties):
```

Comment:
What's the difference between `rd_lvl` and `rf_lvl`?

Reply:
I'm not sure. `rf_lvl` is redundancy factor level. I asked the person who added `rd_lvl` here (Xuezhao) and he sounded like it's okay to add this additional condition.